doc/acting/discontinued/teb/motion_planner.py

```
@@ -0,0 +1,1028 @@
+from geometry_msgs.msg import Point, Pose, PoseStamped, Quaternion, Point32
+from nav_msgs.msg import Path
+from perception.msg import LaneChange, Waypoint
+from ros_compatibility.node import CompatibleNode
+from rospy import Publisher, Subscriber, ServiceProxy
+from scipy.spatial.transform import Rotation
+from std_msgs.msg import Bool, Float32, Float32MultiArray, Int16, String, Empty
+from scipy.spatial.transform import Rotation as R
+
+from utils import (
+    NUM_WAYPOINTS,
+    NUM_WAYPOINTS_OVERTAKE,
```

Ruff (0.8.2)

`utils.NUM_WAYPOINTS_OVERTAKE` imported but unused

Remove unused import

(F401)